examples/basic-checkpoint-ha.yaml

```diff
@@ -40,7 +40,7 @@ spec:
       cpu: 1
   podTemplate:
     spec:
-      serviceAccount: flink-operator
+      serviceAccount: flink
```

Comment:
We could also update the serviceAccount in `e2e-tests/data/cr.yaml` to `flink`.

Reply:
Yes, great catch.